tests/test_fit_phot_model.py

```diff
@@ -0,0 +1,79 @@
+# LSST Data Management System
+# Copyright 2017 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+
+from __future__ import print_function, absolute_import, division
+
+import numpy as np
+import os
+
+from lsst.validate.drp.photerrmodel import photErrModel, fitPhotErrModel
+
+sigmaSys, gamma, m5 = 0.01, 0.039, 24.35  # mag, '', mag
+# m = randn(100)*2 + 25
```

Comment:
What are these commented lines for? Can they go away?

Now that I look further, maybe they're how the data below was generated? If so, please say so with a sentence or two. And maybe generate them directly that way, instead of having them typed in here?